clang-format-changed-files.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
```

Comment:
Can't tell on github UI but please make that the `chmod u+x` for this file is part of the commit.

Reply:
Yes, we did that.